Sources/Core/Int.swift

```diff
@@ -8,22 +8,38 @@
 @frozen
 public struct Int {
   @usableFromInline
-  internal var _value: Builtin.Int32
+  internal var _value: Builtin.Word
```

Comment:
While the change is correct, this is not related to the handling of `&` operations.